apps/book/models.py

```diff
@@ -62,15 +62,15 @@ class LanguageType(models.TextChoices):
         NEPALI = 'nepali', _('Nepali')
         MAITHALI = 'Maithali', _('Maithali')
         THARU = 'Tharu', _('Tharu')
-        BILINGUAL = 'bilingual', _('BILINGUAL')
+        BILINGUAL = 'bilingual', _('Bilingual')
 
     class Grade(models.TextChoices):
         ECD = 'ecd', _('ECD')
         GRADE_1 = 'grade_1', _('Grade 1')
         GRADE_2 = 'grade_2', _('Grade 2')
         GRADE_3 = 'grade_3', _('Grade 3')
-        GRADE_4 = 'grade_4', _('Grade 4')
-        GRADE_5 = 'grade_5', _('Grade 5')
+        # GRADE_4 = 'grade_4', _('Grade 4')
+        # GRADE_5 = 'grade_5', _('Grade 5')
```

Comment:
Can you confirm is this willn't break according to existing data?

Reply:
According to @kopitek8  there are no books of grade 4,  5 now in production server, and for this season we don't upload books for grade 4,5 

@kopitek8  Can you confirm here too?

Reply:
@bimal125 Can you also do a quick query and confirm this as well?

Reply:
No books for grade 4 & 5. Confirmed